themes/docsy/layouts/learningpaths/baseof.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
```

Comment:
Were you experimenting with a learningpaths layout? Looks like there's no content section yet so maybe we shouldn't include these files in this PR just yet?

Reply:
Gah must have committed too much, I excluded most of it. Will fix.